stdlib/ast.pyi

```diff
@@ -168,7 +168,7 @@ class NodeTransformer(NodeVisitor):
     #       The usual return type is AST | None, but Iterable[AST]
     #       is also allowed in some cases -- this needs to be mapped.
 
-_T = TypeVar("_T", bound=AST)
+_T = _TypeVar("_T", bound=AST)
```

Comment:
Does pytype like it any more if we do `_T = typing.TypeVar("_T")`? (FWIW that would also be friendlier to flake8-pyi, which will recognise `typing.TypeVar("_T")` as a TypeVar declaration, but won't recognise `_TypeVar("_T")` as a TypeVar declaration.)